.rhcicd/frontend.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: notifications-frontend
+objects:
+  - apiVersion: cloud.redhat.com/v1alpha1
+    kind: Frontend
+    metadata:
+      name: notifications
+    spec:
+      envName: ${ENV_NAME}
+      title: notifications
+      deploymentRepo: https://github.com/RedHatInsights/notifications-frontend
+      API:
+        versions:
+          - v1
+      frontend:
+        paths:
+          - /apps/notifications
+      image: ${IMAGE}:${IMAGE_TAG}
+      navItems:
+        - appId: "notifications"
+          title: "notifications"
+          href: "/settings/notifications"
+          product: "Red Hat Insights"
```

Comment:
This app also serves `/settings/integrations`

Reply:
We'll add `integrations` as well. I thought it was a separate app. As for the submenus, that will get updated in another central PR since the nav gets shared by other resources.